packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -217,8 +268,13 @@ export abstract class FluidDataStoreContext
 		return this.parentContext.baseLogger;
 	}
 
+	private readonly _deltaManager: ContextDeltaManagerProxy;
 	public get deltaManager(): IDeltaManager<ISequencedDocumentMessage, IDocumentMessage> {
-		return this.parentContext.deltaManager;
+		return this._deltaManager;
+	}
+
+	public get readonly(): boolean | undefined {
+		return this.parentContext.readonly;
```

Comment:
Want to assert that `this._deltaManager.readonlyInfo.readonly` agrees?

Reply:
no, as i plan to remove the agreement shortly

Reply:
Even after we start simulating Readonly, the `ContextDeltaManagerProxy` should agree, right?